src/main/java/io/gravitee/repository/jdbc/management/JdbcCommandRepository.java

```diff
@@ -94,10 +92,10 @@ protected String getId(Command item) {
     public Optional<Command> findById(String id) throws TechnicalException {
         LOGGER.debug("JdbcCommandRepository.findById({})", id);
         try {
-            JdbcHelper.CollatingRowMapper<Command> rowMapper = new JdbcHelper.CollatingRowMapper<>(ORM.getRowMapper(), CHILD_ADDER, "id");
-            jdbcTemplate.query("select * from " + escapeReservedWord("commands") + " c " +
-                            "left join command_acknowledgments ca on c.id = ca.command_id " +
-                            "left join command_tags ct on c.id = ct.command_id " +
+            JdbcHelper.CollatingRowMapper<Command> rowMapper = new JdbcHelper.CollatingRowMapper<>(getOrm().getRowMapper(), CHILD_ADDER, "id");
+            jdbcTemplate.query("select * from " + escapeReservedWord(this.tableName) + " c " +
```

Comment:
escape table name in the constructor
```suggestion
            jdbcTemplate.query(getOrm().getSelectAllSql() + " c " +
```